File: app/src/main/java/fr/iut/iem/pokecard/data/manager/PokeDB.java
package fr.iut.iem.pokecard.data.manager;

import com.raizlabs.android.dbflow.annotation.Database;

@Database(name = PokeDB.DB_NAME, version = PokeDB.VERSION)
public class PokeDB {
    public static final String DB_NAME = "Poke";
    public static final int VERSION = 1;
}

File: app/src/main/java/fr/iut/iem/pokecard/data/manager/PokeDB.kt
package fr.iut.iem.pokecard.data.manager

import com.raizlabs.android.dbflow.annotation.Database

@Database(name = PokeDB.DB_NAME, version = PokeDB.VERSION)
class PokeDB {
    companion object {
        const val DB_NAME = "Comics"
        const val VERSION = 1
    }
}